src/crypto.rs

```diff
@@ -98,6 +101,40 @@ impl SecretKey {
     }
 }
 
+impl Sized for SecretKey {
```

Comment:
why do we need these `impl`s for `SecretKey`?

Reply:
fixed